pypesto/ensemble/ensemble.py

```diff
@@ -327,8 +328,45 @@ def __init__(self,
             self.predictions = predictions
 
     @staticmethod
-    def from_sample(result: Result, chain_index: int = 0, **kwargs):
-        x_vectors = result.sample_result.trace_x[chain_index].T
+    def from_sample(
+            result: Result,
+            chain_index: int = 0,
+            remove_burn_in: bool = True,
+            chain_slice: slice = None,
+            **kwargs,
+    ):
+        """Construct an ensemble from a sample.
+
+        Parameters
+        ----------
+        result:
+            A pyPESTO result that contains a sample result.
+        chain_index:
```

Comment:
Does this argument `chain_index` make sense for calculation of prediction intervals?
I think one should always examine the chain with the lowest temperature... That would be the closest to the true posterior.

Any thoughts @yannikschaelte ?

Reply:
predictions make only sense w.r.t. T=1, that is right. only reasons to have this index would be if it is good for some kind of debugging ... or checking what happens if the likelihood noise is increased.

Reply:
> checking what happens if the likelihood noise is increased.

the likelihood noise is not taking place in the state predictions, only in the validation intervals, no?

Reply:
I meant T>1 effectively translates to running things with a higher noise variance, therefore all downstream analyses kind of assume that too ...